src/OSPSuite.Core/Domain/DistributedParameter.cs

```diff
@@ -132,9 +132,14 @@ public override bool IsFixedValue
          set
          {
             //percentile set first so that correct value is available if a value event is raised
-            PercentileParameter.IsFixedValue = value;
+            //Note: Percentile parameter may be null when constructing the distributed parameter dynamically
+            if (PercentileParameter != null)
+               PercentileParameter.IsFixedValue = value;
+
             base.IsFixedValue = value;
-            OnPropertyChanged(() => Percentile);
+
+            if (PercentileParameter != null)
```

Comment:
This is required to be able to solve all those small issues in PKSim. Merging. Please review
I don't want to check everywhere. Once one manipulated the parameter, those sub parameters SHOULD BE defined. This is just a weird use case of overriding the formula with a table formula aging